src/views/factor-verify/EmailMagicLinkForm.js

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+/* eslint complexity: [2, 7] */
+define(['okta', 'q', 'views/mfa-verify/PassCodeForm'], function (Okta, Q, PassCodeForm) {
+
+  var subtitleTpl = Okta.Handlebars.compile('({{subtitle}})');
+  var _ = Okta._;
+  var API_RATE_LIMIT = 30000; //milliseconds
+
+  return PassCodeForm.extend({
```

Comment:
probably we shouldn't extend `PassCodeForm` since this form doesn't need a passcode. `PassCodeForm` doesn't have any functions/attributes that we can use here, so it is probably better to extend `Okta.Form.extend`